Library/Formula/path-extractor.rb

```diff
@@ -0,0 +1,28 @@
+require "language/go"
+
+class PathExtractor < Formula
+  desc "unix filter which outputs the filepaths found in stdin"
+  homepage "https://github.com/edi9999/path-extractor"
+  url "https://github.com/edi9999/path-extractor/archive/v0.1.0.tar.gz"
+  sha256 "33fe041196a36161a67cddb20405ad9d53c9b6fba4f30b8e6bc6c3e1ce0ac1c8"
+
+  head "https://github.com/edi9999/path-extractor.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOBIN"] = bin
+    ENV["GOPATH"] = buildpath
+    ENV["GOHOME"] = buildpath
+
+    mkdir_p buildpath/"src/github.com/edi9999/"
+    ln_sf buildpath, buildpath/"src/github.com/edi9999/path-extractor"
+    Language::Go.stage_deps resources, buildpath/"src"
```

Comment:
There aren't any resources being used here. Perhaps there should be if it has any go dependencies?